### ISSUE TITLE
Allow user-specified metric to be used with conditional metrics

### ISSUE THREAD

**Issue by dzeber** [mozilla/PRESC#258]
Currently the metric used in the [conditional metric](https://github.com/mozilla/PRESC/blob/master/presc/evaluations/conditional_metric.py#L111) evaluation is hardcoded. We should allow one or more metrics to be specified by the user. A user-specified metric should include both the function and a display name.

**Comment by dzeber** [comment]
I was thinking we allow a list of metrics in the config, eg.
```yaml
evaluations:
  conditional_metric:
    metrics:
      - function: accuracy_score
        display_name: Accuracy   # optional
```
If there are multiple metrics, the simplest thing is just to rerun the [computation/display](https://github.com/mozilla/PRESC/blob/master/presc/evaluations/conditional_metric.py#L177).

We should also allow for the metrics to be set in the column-specific overrides which get picked up in [`compute_for_column`](https://github.com/mozilla/PRESC/blob/master/presc/evaluations/conditional_metric.py#L127), eg. 
```yaml
evaluations:
  conditional_metric:
    computation:
      columns:
        my_col_name:
          num_bins: 5
          metrics:
            - function: accuracy_score
              display_name: Accuracy   # optional
```

For an initial version, we can just restrict the `function` to be the name of a scikit-learn [metric](https://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics) if that makes life easier. Not sure what the considerations would be to allow for a user-coded function. Eg., would we want to allow `function` to be a string of Python code, or an import statement?
